cmake/FindCUDA/FindCUDA/run_nvcc.cmake

```diff
@@ -175,7 +175,7 @@ endif()
 
 # nvcc doesn't define __CUDACC__ for some reason when generating dependency files.  This
 # can cause incorrect dependencies when #including files based on this macro which is
-# defined in the generating passes of nvcc invokation.  We will go ahead and manually
+# defined in the generating passes of nvcc invocation.  We will go ahead and manually
```

Comment:
How should we handle PRs to external libs like FindCUDA? Modifying it here will lead to divergence from the master branch, except if we send the PR there.
This has already happened in https://github.com/pytorch/pytorch/pull/2254 for example